mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -89,9 +84,8 @@ CLASS com/mojang/blaze3d/systems/RenderSystem com/mojang/blaze3d/systems/RenderS
 	METHOD enableBlend enableBlend ()V
 	METHOD enableColorLogicOp enableColorLogicOp ()V
 	METHOD enableCull enableCull ()V
-	METHOD enableDepthTest enableDepthTest ()V
 	METHOD enablePolygonOffset enablePolygonOffset ()V
-	METHOD enableScissor enableScissor (IIII)V
+	METHOD enableScissor (IIII)V
```

Comment:
The parameters on this swap the width and height. That seems wrong?

Reply:
reset all of them to be proposed based on the gl method

Reply:
Thanks. Looked weird from a quick glance